deploy/helm/kubecf/values.yaml

```diff
@@ -238,6 +238,12 @@ features:
     registry:
       service:
         nodePort: 32123
+    rbac:
+      cluster_role:
+        eirini_webhook:
```

Comment:
+1 on being specific here on the `eirini_webhook`. I'd swap it with the `cluster_role` level:

```yaml
rbac:
  eirini_webhook:
    cluster_role:
      ...
```

Reply:
This is a good idea. Thanks, change it to: 
```
cluster_role:
  rules: 
......
```